src/DynamoCore/Graph/Nodes/PortModel.cs

```diff
@@ -505,6 +505,31 @@ internal string GetOutPortType()
         }
     }
 
+    /// <summary>
+    /// This currently doesn't do anything other than copy the portmodel it receives.
+    /// Keeping it here for now as we might add to it when a decision about how to 
+    /// move forward has been made.
+    /// </summary>
+    internal class ProxyPortModel : PortModel
+    {
+        //private int index;
+        //private Point2D center;
+
+        public ProxyPortModel(PortModel portModel) : base(portModel.PortType, portModel.Owner, new PortData(portModel.Name, portModel.ToolTip))
+        {
+            //this.index = portModel.Index;
+        }
+
+        //public override int Index { get => index; }
```

Comment:
Delete commented code.

Reply:
Yea for this one im keeping all of this for now, simply because theres another task to figure out how we want to handle this as the comment describes this dosent really do anything for now but might be useful for later